app/scripts/controllers/app-state.js

```diff
@@ -15,28 +15,42 @@ export default class AppStateController extends EventEmitter {
       showUnlockRequest,
       preferencesStore,
     } = opts
-    const { preferences } = preferencesStore.getState()
+    const { preferences, selectedAddress } = preferencesStore.getState()
 
     super()
 
     this.onInactiveTimeout = onInactiveTimeout || (() => {})
-    this.store = new ObservableStore(Object.assign({
-      timeoutMinutes: 0,
-      connectedStatusPopoverHasBeenShown: true,
-    }, initState))
+    this.store = new ObservableStore(
+      Object.assign(
+        {
+          connectedStatusPopoverHasBeenShown: true,
+          timeoutMinutes: 0,
+        },
+        initState,
+        {
+          switchToConnectedAlertShown: {},
```

Comment:
I think this PR is fine to merge as-is, but it might be better to move this stuff into the new `AlertController` created in #8550. If so, maybe it'd be better to get #8550 merged first. Either way I move this afterwards in a separate PR though.

Reply:
This was done in 55f4b3e22e2d2fd63d92dcce09b5b570d875e9fd